app/views/puppetdb_foreman/nodes/show.html.erb

```diff
@@ -0,0 +1,53 @@
+<% title _('PuppetDB Node') %>
+
+<%= host_breadcrumb %>
```

Comment:
This needs foreman >= 1.18. Can you please either add a guard (`respond_to?(: host_breadcrumb)`) for this or bump the required foreman version in `lib/puppetdb_foreman/engine.rb`?
Also, you can omit the `=` here. `<% host_breadcrumb` should be enough.